scripts/test-e2e-standalone.sh

```diff
@@ -58,4 +58,3 @@ fi
 
 echo Done waiting for OLS server start, running e2e
 make test-e2e
```

Comment:
`set -e` at beginning of script might do the magic

Reply:
yeah i considered it and may still but i've been burned before by the surprising effects of set -e when you don't want them. For now this should be sufficient i think.